var/spack/repos/builtin/packages/metis/package.py

```diff
@@ -208,10 +211,12 @@ def install(self, spec, prefix):
             make()
             make('install')
 
-            # install GKlib headers, which will be needed for ParMETIS
-            gklib_dist = join_path(prefix.include, 'GKlib')
-            mkdirp(gklib_dist)
-            install(join_path(source_directory, 'GKlib', '*.h'), gklib_dist)
+            # install all headers, which will be needed for ParMETIS and other programs
+            subdirs = ["GKlib", "libmetis", "programs"]
```

Comment:
Are all these headers available in all versions of metis? and it seems like "programs" wouldn't be installed to the include dir...

Reply:
We are already under:
```python
@when('@5:')
def install(self, spec, prefix):
```

This `programs` directory only contains headers files (`.h`).

Reply:
Ah nice, I didn't see that. Thanks!